### PR TITLE
Milestone 1: Design Teardown Project is finished

### PR DESCRIPTION
**Pull request by RaminMammadzada**
In this project:
- The header, body and footer parts are created.
- Used float, flex and grid.
- Used shadow and gradients.
- HTML and CSS animation features are used.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+			<div class="more-button">
+				<button style="padding-right: 0px;" class="buIcon" type="submit">More </button><img class="menu-cat-img"
+					src="images/menu-cat.svg" alt="">
+			</div>
+			<div class='formContainer'>
+				<form>
+					<div class='inputWhite'><i class="fa fa-search fa-lg" aria-hidden="true"></i><input
+							placeholder="search articles..." class="searchForm" type='text'></div>
+				</form>
+			</div>
```
Comment (elmejdki) [inline index.html]: please use in the nav a ul li element

File: index.html [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+					</div>
+				</div>
+				<h1>Solving Common Cross-Platform Issues When Working with Flutter 
+					<span class='inline'><i class="fa fa-envelope-o" aria-hidden="true"></i> Comment</span>
+				</h1>
+				
+				<div class='spanContainer'><span>Smashy Monthly Roundup<sup> 9</sup></span></div>
+				<i class="fa fa-long-arrow-right fa-3x" aria-hidden="true"></i>
+			</div>
+			<div style="background: #858585;" class='profileCon'>
```
Comment (elmejdki) [inline index.html]: please remove all the inline styles from your HTML and put it in CSS files

File: index.html [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+
+				</div>
+				<h1>What Vitruvius Can Teach Us About Web Design <span class='inline'><i class="fa fa-envelope-o"
+							aria-hidden="true"></i> Comment</span>
+				</h1>
+				<div class='spanContainer'><span># React<sup> 45</sup> </span><span># Javascript<sup>
+							274</sup></span><span># Tools<sup> 229</sup></span></div>
+				<i class="fa fa-long-arrow-right fa-3x" aria-hidden="true"></i>
+			</div>
+			<div style="background: #757575;" class='profileCon'>
```
Comment (elmejdki) [inline index.html]: remove the inline style and put it in your CSS file

**Review by elmejdki — CHANGES_REQUESTED** [review]
# Great job guys so far :clap:, your page looks really similar to the original page but in order to get it approved you need to make some changes.
### status: require change

- please add a .gitignore file and mention this .DS_STORE file to remove it from the repo:
image
- please use the [HTML validator](www.validator.w3.org) to validate your HTML, and fix all the errors:
image
- please use article tags in your project:
image

### Please make all the required changes before you submit another code review.
### Happy Coding :tophat:

File: index.html (hunk 1 above)
Comment (RaminMammadzada) [reply index.html]: The changes are implemented.

File: index.html (hunk 2 above)
Comment (RaminMammadzada) [reply index.html]: The changes are implemented.

File: index.html (hunk 3 above)
Comment (RaminMammadzada) [reply index.html]: The changes are implemented.

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Nice work team 👏👏!

### But we need to work on some issues before we can merge.

- There is a horizontal scrollbar in your clone. Check the inline comments for more details.
Annotation 2020-06-23 115739

- Your clone looks amazing but you have strayed away from the requirements of the project. The objective is to understand where and how each section fits inside a webpage. Not making the actual clone. The requirement was to make a heatmap of the original page. You were required to use grayscale color (black, white, or any shade of gray) to indicate each element including images. I see that you have used grayscale in a few places. Please do that to all the other places. Remove the images and instead use placeholders (a grayscale box indicating that there is supposed to be an image). Remember to use grayscale colors.

image

image

image

image

image

image

- The footer needs to have two image placeholders on either side.

#### Clone
Annotation 2020-06-23 120928

#### Original
Annotation 2020-06-23 120947

- Your CSS has invalid properties. Check the comments below.

File: style/styles.css [hunk 4]
```diff
@@ -0,0 +1,1101 @@
...
+  height: 400px;
+  color: white;
+  width: 86%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.featuredBookText {
+  margin-top: 400px;
+  flex: 70%;
```
Comment (tirthajyoti-ghosh) [inline style/styles.css:848]: This could be a potential cause for the horizontal scrollbar in your clone.

File: style/styles.css [hunk 5]
```diff
@@ -0,0 +1,1101 @@
...
+  color: rgb(71, 68, 68);
+  margin-left: 5px;
+}
+
+.redArrow {
+  text-align: center;
+  width: 150px;
+  color: rgb(158, 19, 19);
+  padding-bottom: 30px;
+  padding-top: -20px;
```
Comment (tirthajyoti-ghosh) [inline style/styles.css]: No `padding` property accepts negative values. This is invalid.

File: style/styles.css [hunk 6]
```diff
@@ -0,0 +1,1101 @@
...
+  color: white;
+  padding: 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  margin: 4px 2px;
+  cursor: pointer;
+  border-radius: 12px;
+  font-size: 100%;
+  text-shadow: #1f2c43;
```
Comment (tirthajyoti-ghosh) [inline style/styles.css]: You have only added color to the `text-shadow`. But it needs two more values -> `h-shadow` and ` v-shadow`. 
https://www.w3schools.com/cssref/css3_pr_text-shadow.asp

File: style/styles.css [hunk 7]
```diff
@@ -0,0 +1,1101 @@
...
+  color: #277ccc;
+  padding: 20px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  margin: 4px 2px;
+  cursor: pointer;
+  border-radius: 12px;
+  font-size: 100%;
+  text-shadow: #1f2c43;
```
Comment (tirthajyoti-ghosh) [inline style/styles.css]: You have only added color to the `text-shadow`. But it needs two more values -> `h-shadow` and ` v-shadow`. 
https://www.w3schools.com/cssref/css3_pr_text-shadow.asp

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Remember to use grayscale colors for the heatmap.
image

Apply these changes and request another review.

Happy coding 👨‍💻

File: style/styles.css (hunk 4 above)
Comment (RaminMammadzada) [reply style/styles.css:848]: It is all done.

File: style/styles.css (hunk 5 above)
Comment (RaminMammadzada) [reply style/styles.css]: Changes implemented.

File: style/styles.css (hunk 6 above)
Comment (RaminMammadzada) [reply style/styles.css]: Changes implemented.

File: style/styles.css (hunk 7 above)
Comment (RaminMammadzada) [reply style/styles.css]: Changes implemented.

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle: 

### Hi @RaminMammadzada and @Alexoid1   :hand: 

### Great project so far, but there is some changes to be made:
- We still have the horizontal scroll bar, as requested by the previous reviewer, you need to remove that.
Screenshot from 2020-06-23 10-42-04
- There some invalid css properties that you  used, the previous TSE asked to fix, please address that:
Screenshot from 2020-06-23 10-43-25

Screenshot from 2020-06-23 10-43-10

- You did not configure the Github actions correctly, go to this link then read and implement all the steps https://github.com/microverseinc/linters-config/tree/master/html-css. I don't see all the files in your project (hintrc, stylelintrc):
Screenshot from 2020-06-23 10-46-33
### once done with the changes, please ask for another code review

**Comment by RaminMammadzada** [comment]
> ### Status: Change Required ♻️
> ### Hi @RaminMammadzada and @Alexoid1 ✋
> ### Great project so far, but there is some changes to be made:
> * We still have the horizontal scroll bar, as requested by the previous reviewer, you need to remove that.
>   Screenshot from 2020-06-23 10-42-04
> * There some invalid css properties that you  used, the previous TSE asked to fix, please address that:
>   Screenshot from 2020-06-23 10-43-25
> 
> Screenshot from 2020-06-23 10-43-10
> 
> * You did not configure the Github actions correctly, go to this link then read and implement all the steps https://github.com/microverseinc/linters-config/tree/master/html-css. I don't see all the files in your project (hintrc, stylelintrc):
>   Screenshot from 2020-06-23 10-46-33
> 
> ### once done with the changes, please ask for another code review

I Changed them already.

**Comment by RaminMammadzada** [comment]
> ### Status: Change Required ♻️
> ### Hi @RaminMammadzada and @Alexoid1 ✋
> ### Great project so far, but there is some changes to be made:
> * We still have the horizontal scroll bar, as requested by the previous reviewer, you need to remove that.
>   Screenshot from 2020-06-23 10-42-04
> * There some invalid css properties that you  used, the previous TSE asked to fix, please address that:
>   Screenshot from 2020-06-23 10-43-25
> 
> Screenshot from 2020-06-23 10-43-10
> 
> * You did not configure the Github actions correctly, go to this link then read and implement all the steps https://github.com/microverseinc/linters-config/tree/master/html-css. I don't see all the files in your project (hintrc, stylelintrc):
>   Screenshot from 2020-06-23 10-46-33
> 
> ### once done with the changes, please ask for another code review

I have already solved the shadow problems. You can check the commits before your messages and in live demo.

**Review by trekab — APPROVED** [review]
## Approved! :tada: 
Hi @RaminMammadzada and @Alexoid1! Well done! :+1:  You have met all the project requirements. You may merge to master, submit the project completion form, and proceed to your next project.

_Happy coding!_ :wink: 
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Pull request merged**